api/model/src/test/java/org/projectnessie/model/TestModelObjectsSerialization.java

```diff
@@ -65,211 +69,204 @@ static List<Case> goodCases() {
+    Object obj;
+    Class<?> serializationView;
     final Class<?> deserializeAs;
-    final String deserializedJson;
-
-    public Case(Class<?> deserializeAs, Json deserializedJson) {
-      this(null, deserializeAs, deserializedJson);
-    }
+    ObjectNode deserializedJson = JsonNodeFactory.instance.objectNode();
+    boolean skipFinalCompare;
 
-    public Case(Object obj, Class<?> deserializeAs, Json deserializedJson) {
-      this(obj, Views.V1.class, deserializeAs, deserializedJson);
-    }
-
-    public Case(
-        Object obj, Class<?> serializationView, Class<?> deserializeAs, Json deserializedJson) {
-      this.obj = obj;
-      this.serializationView = serializationView;
+    public Case(Class<?> deserializeAs) {
       this.deserializeAs = deserializeAs;
-      this.deserializedJson = deserializedJson.toString();
     }
 
-    @Override
-    public String toString() {
-      return deserializeAs.getName() + " : " + obj;
-    }
-  }
-
-  protected static
-  class Json { // Helps in building json strings, which can be used for verification.
-
-    @SuppressWarnings("InlineFormatString")
-    private static final String STR_KV_FORMAT = "%s,\"%s\":\"%s\"";
-
-    @SuppressWarnings("InlineFormatString")
-    private static final String NO_QUOTES_KV_FORMAT = "%s,\"%s\":%s";
-
-    String currentContent;
-
-    private Json(String currentContent) {
-      this.currentContent = currentContent;
-    }
-
-    public static Json from(String key, String val) {
-      return new Json(String.format("\"%s\":\"%s\"", key, val));
-    }
-
-    public static Json noQuotes(String key, Object val) {
-      return new Json(String.format("\"%s\":%s", key, val));
-    }
-
-    public static Json arr(String key, String... val) {
-      String currentContent =
-          Stream.of(val)
-              .collect(Collectors.joining("\",\"", String.format("\"%s\":[\"", key), "\"]"));
-      return new Json(currentContent);
-    }
-
-    public Json add(String key, String val) {
-      this.currentContent = String.format(STR_KV_FORMAT, currentContent, key, val);
+    public Case obj(Object obj) {
+      this.obj = obj;
       return this;
     }
 
-    public Json addArr(String key, String... val) {
-      String keyContent =
-          Stream.of(val)
-              .map(v -> String.format("\"%s\"", v))
-              .collect(Collectors.joining(",", String.format("\"%s\":[", key), "]"));
-      this.currentContent = String.format("%s,%s", currentContent, keyContent);
+    public Case view(Class<?> serializationView) {
+      this.serializationView = serializationView;
       return this;
     }
 
-    public Json addArrNoQuotes(String key, Object... val) {
-      String keyContent =
-          Stream.of(val)
-              .map(Object::toString)
-              .collect(Collectors.joining(",", String.format("\"%s\":[", key), "]"));
-      this.currentContent = String.format("%s,%s", currentContent, keyContent);
+    public Case skipFinalCompare() {
```

Comment:
Currently unused. I suppose this planned for #6480? Same for the `view` method above.

Reply:
Ah, yes. That'll be used in that PR.